### PR TITLE
[IOS-3007]Add more logs for Adapter

### PR DESCRIPTION
**Pull request by YueVungle**
MoPub side suggested us that adding adapter logging for Banner adapter.
This PR adds more logs for MoPub Adapter.

IOS-3007

### CODE REVIEW

File: Vungle/VungleRouter.m [hunk 1]
```diff
@@ -653,14 +657,17 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
 
     if ([self.delegatesDict objectForKey:placementID]) {
         if (isAdPlayable) {
+            MPLogInfo(@"Ad playability update returned ad is playable for Placement ID: %@", placementID);
```
Comment (ashinagawa) [inline Vungle/VungleRouter.m]: You might want to add "Vungle: " here as well?

File: Vungle/VungleRouter.m [hunk 2]
```diff
@@ -681,7 +689,9 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
                             MPLogInfo(@"Vungle: Ad playability update returned error for Placement ID: %@, Error: %@", placementID, error.localizedDescription);
                             playabilityError = error;
                         } else {
-                            playabilityError = [NSError errorWithCode:MOPUBErrorAdapterFailedToLoadAd localizedDescription:@"Vungle: Ad playability update returned Ad is not playable."];
+                            NSString *message = [NSString stringWithFormat:@"Vungle: Ad playability update returned Ad is not playable for Placement ID: %@.", placementID];
```
Comment (ashinagawa) [inline Vungle/VungleRouter.m:692]: You might want to add "Vungle: " here as well?

File: Vungle/VungleRouter.m [hunk 3]
```diff
@@ -673,6 +680,7 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
             for (int i = 0; i < self.bannerDelegates.count; i++) {
                 if (([[(id<VungleRouterDelegate>)[self.bannerDelegates[i] valueForKey:kVungleBannerDelegateKey] getPlacementID] isEqualToString:placementID]) && ((BannerRouterDelegateState)[[self.bannerDelegates[i] valueForKey:kVungleBannerDelegateStateKey] intValue] == BannerRouterDelegateStateRequesting)) {
                     if (isAdPlayable) {
+                        MPLogInfo(@"Ad playability update returned ad is playable for Placement ID: %@", placementID);
```
Comment (ashinagawa) [inline Vungle/VungleRouter.m]: You might want to add "Vungle: " here as well?

**Review by ashinagawa — COMMENTED** [review]
I think we need to add "Vungle: " for all `MPLogEvent` to be consistent if there is any other place missing this?

File: Vungle/VungleRouter.m (hunk 2 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m:692]: @ashinagawa Got it, thanks a lot for the good suggestion. :)

File: Vungle/VungleRouter.m (hunk 3 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m]: @ashinagawa Got it, thanks a lot :)

File: Vungle/VungleRouter.m (hunk 1 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m]: @ashinagawa Got it, thanks a lot for the good suggestion. :)

**Comment by YueVungle** [comment]
> I think we need to add "Vungle: " for all `MPLogEvent` to be consistent if there is any other place missing this?

@ashinagawa Hi Aki, I will check all the MoPub log to add "Vungle: "  to keep consistency, thanks a lot for the guidance.  ^_^

File: Vungle/VungleRouter.m (hunk 2 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m:692]: @ashinagawa Hi Aki, the `message` string is already started with "Vungle: " here. Thanks ^_^

File: Vungle/VungleRouter.m (hunk 1 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m]: @ashinagawa  Updated, thanks ^_^

File: Vungle/VungleRouter.m (hunk 3 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m]: @ashinagawa  Updated, thanks ^_^

**Comment by YueVungle** [comment]
> > I think we need to add "Vungle: " for all `MPLogEvent` to be consistent if there is any other place missing this?
> 
> @ashinagawa Hi Aki, I will check all the MoPub log to add "Vungle: " to keep consistency, thanks a lot for the guidance. ^_^

@ashinagawa Hi Aki, I found some logs did not add "Vungle: " , added. Thank you for the good catch. 👍

**Review by crbinsf — APPROVED** [review]
👍

**Review by ashinagawa — APPROVED** [review]
LGTM

**Pull request merged**